### ISSUE TITLE
Kaydle is mentioned in the README but the only version has been yanked.

### ISSUE THREAD

**Issue by alilleybrinker** [kdl-org/kdl-rs#42]
As the title says, it looks like there are no available versions of Kaydle on Crates.io. Should the mention be removed unless/until a new version is published?

**Comment by zkat** [comment]
@Lucretiel do you have any thoughts?

**Comment by Lucretiel** [comment]
`kaydle` is not ready for use; I published it to crates.io only to reserve the name. The crate that's there doesn't have any unsoundness but does have large swaths of unavailable / incomplete functionality, which is why I yanked it. I've been making steady progress on it but it'll be weeks at the minimum and likely months before I'll have something releasable, so I don't mind at all if you'd like to strike it from the KDL docs until that time comes.

There *is* a stable crate called `kaydle-primitives` that contains all the parsing logic used by `kaydle`, but it's all highly generic / skeletal (since it's intended to be used by kaydle) so it's probably not a good choice for end-user applications. That being said, I am sticking to semver for it in case anyone does want to use it.
